app/templates/base.html

```diff
@@ -47,6 +47,31 @@ <h2>Yalies</h2>
             {% endblock %}
         </main>
         <footer>
+            <button class="scrollToTopBtn">⇪</button>
+            <script>
+                var target = document.getElementById('navbar');
+                var scrollToTopBtn = document.querySelector(".scrollToTopBtn")
```

Comment:
Use single quotes for all JavaScript code, don't switch between single and double